### PR TITLE
fix exercise 3 sample yaml

### PR DESCRIPTION
**Pull request by mk2**
I have faced some invalid sample yaml codes in exercise 3. So, fixed them.
1. `apache_webserver_port` is missing. (but used in the template)
2. `{% raw %}` is not needed.

When I have learned ansible by the lightbulb at Redhat JP ([Ansible MokuMoku Kai #1](https://ansible-users.connpass.com/event/79253/)), found them. Thanks Redhat JP Experts.

### CODE REVIEW

File: guides/ansible_engine/3-variables/README.md [hunk 1]
```diff
@@ -48,22 +48,21 @@ Add a play definition and some variables to your playbook.  These include addtio
       - mod_wsgi
     apache_test_message: This is a test message
     apache_max_keep_alive_requests: 115
+    apache_webserver_port: 80
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: This is a good catch, thanks!

File: guides/ansible_engine/3-variables/README.md [hunk 2]
```diff
@@ -175,6 +173,7 @@ Your new, improved playbook is done! Let's take a second look to make sure every
...
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: Thanks, this is a good catch as well.

File: guides/ansible_engine/3-variables/README.md [hunk 3]
```diff
@@ -48,22 +48,21 @@ Add a play definition and some variables to your playbook.  These include addtio
...
 ```
 
 ### Step 3
 
 Add a new task called *httpd packages are present*:
 
 ```yml
-{% raw %}
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.

File: guides/ansible_engine/3-variables/README.md [hunk 4]
```diff
@@ -48,22 +48,21 @@ Add a play definition and some variables to your playbook.  These include addtio
...
   tasks:
     - name: Ensure httpd packages are present
       yum:
         name: "{{ item }}"
         state: present
       with_items: "{{ httpd_packages }}"
       notify: restart-apache-service
-{% endraw %}
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.

File: guides/ansible_engine/3-variables/README.md [hunk 5]
```diff
@@ -164,7 +163,6 @@ handlers:
 Your new, improved playbook is done! Let's take a second look to make sure everything looks the way you intended.  If not, now is the time for us to fix it up. The figure below shows line counts and spacing.
 
 ```yml
-{% raw %}
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.

File: guides/ansible_engine/3-variables/README.md [hunk 6]
```diff
@@ -211,7 +210,6 @@ Your new, improved playbook is done! Let's take a second look to make sure every
       service:
         name: httpd
         state: restarted
-{% endraw %}
```
Comment (liquidat) [inline guides/ansible_engine/3-variables/README.md]: Unfortunately, as of right now we need the `{% raw %}` tag to be able to export to github pages properly. Without the raw tag, the double curly brackets are not properly formatted.

**Comment by tima** [comment]
@mk2 I agree with @liquidat -- we need those raw tags for (eventually) publishing to Github Pages which we hope to do soon. Please revert those changes in your PR so we can merge the other changes.

**Comment by mk2** [comment]
@tima Thanks reviewing. I have reverted the raw/endraw deletion, and committed.

**Comment by tima** [comment]
Thanks for your help @mk2.

**Pull request merged**